example/src/App.re

```diff
@@ -5,32 +5,17 @@ module Config = {
 
   type routes =
     | Home
-    | Details(string);
+    | Details;
 
-  let routes = [Home, Details("params")];
+  let routes = [Home, Details];
 
   let mapRoute = r =>
     switch (r) {
-    | Home => (
-        "Home",
-        {
-          screen:
-            ReasonReact.wrapReasonForJs(~component=Home.component, Home.make),
-        },
-      )
-    | Details(_) => (
-        "Details",
-        {
-          screen:
-            ReasonReact.wrapReasonForJs(
-              ~component=Details.component,
-              Details.make,
-            ),
-        },
-      )
+    | Home => ("Home", {screen: Home.reactComponent})
```

Comment:
I really like that 😃

Reply:
In the next version we can think about to remove the string from this Variant but for now, it's not that bad :P